identity/identity.go

```diff
@@ -20,6 +20,22 @@ import (
 	"github.com/ory/kratos/x"
 )
 
+// swagger:model identityState
+type State string
+
+func (lt State) IsValid() error {
+	switch lt {
+	case StateActive, StateDisabled:
+		return nil
+	}
+	return errors.New("Identity state is not valid.")
```

Comment:
Errors in Go should be lower cased and no dots :)

```suggestion
	return errors.New("identity state is not valid")
```